rpc/handlers_test.go

```diff
@@ -135,23 +136,23 @@ func TestBlockTransactionCount(t *testing.T) {
 		assert.Equal(t, rpc.ErrBlockNotFound, rpcErr)
 	})
 
-	t.Run("blockId - latest", func(t *testing.T) {
+	t.Run("blockID - latest", func(t *testing.T) {
```

Comment:
These changes should go in the previous commit

Reply:
Done